third_party/xla/xla/tests/BUILD

```diff
@@ -931,6 +931,7 @@ xla_test(
     tags = [
         "optonly",
         "test_xla_cpu_thunks",
+        "cuda-only", #TODO(rocm): weekly sync 24-10-01
```

Comment:
again, only failed subtests instead of the whole test.

Reply:
Fixed as well